src/map/trait.h

```diff
@@ -139,7 +139,11 @@ enum TRAITTYPE
     TRAIT_PRIMEVAL_ZEAL       = 120,
     TRAIT_INSPIRATION         = 121,
     TRAIT_SLEIGHT_OF_SWORD    = 122,
-    TRAIT_DAKEN               = 123
+    TRAIT_DAKEN               = 123,
+    TRAIT_SUPERIOR            = 124, 

```

Comment:
Kj..its a list of strings in that dat and we used an offset to make the first trait "one" and has ZERO TO DO with choosing what is a trait. Likewise I don't give 2 squirts about windowers resource list, they are reading the same strings and deciding anything past a certain offset must be traits. We could easily says traits ended at this value and "gift" started at this next value, internally. Very simply: do they show in game as a trait? Then I am comfy calling it a trait. But please do not make arguments that because the string was in a client file that means something special.